app/src/main/java/ch/epfl/favo/favor/FavorUtil.java

```diff
@@ -48,29 +53,37 @@ public void postFavor(Favor favor) throws RuntimeException {
     }
   }
 
+  @RequiresApi(api = Build.VERSION_CODES.N)
```

Comment:
I think you can just add a supressLint anotation to the class instead of having to annotate each method with @RequiresApi which looks messy

Reply:
Fixed